python/dolfinx/geometry.py

```diff
@@ -270,3 +270,32 @@ def compute_distance_gjk(
 
     """
     return _cpp.geometry.compute_distance_gjk(p, q)
+
+def determine_point_ownership(
+    mesh: Mesh,
+    points: npt.NDArray[np.floating], 
+    cells: typing.Optional[np.ndarray] = None,
```

Comment:
Use npt not np for typing.

Reply:
I noticed np was used in a few other annotations, I added a commit changing them to npt